sfy-buoy/sfy-artemis/src/main.rs

```diff
@@ -303,7 +303,7 @@ fn main() -> ! {
 
         // XXX: This needs to be adapted to frequency, and queue length. Maybe just remove when we
         // have the remaining space check? Check after Hjeltefjorden deployment.
-        const LOOP_DELAY: u32 = 14 * 40_000;
+        const LOOP_DELAY: u32 = 5 * 40_000;
```

Comment:
Don't change the number based on the queue length, change the interval from 40_000 to 20_000. The first number is to make sure there's enough space in the queue, the other to the length it takes to fill up a batch of samples. So the second is tied to the sample rate and should be modified if that is the reason.